using-timescaledb/continuous-aggregates.md

```diff
@@ -248,6 +244,111 @@ hypertable with a continuous aggregate.
 
 ---
 
+### Advanced Topics [](advanced-usage)
+
+#### Continuous Aggregates using Integer-Based Time [](create-integer)
+
+Usually, continuous aggregates are defined on a [date/time-type](https://www.postgresql.org/docs/current/datatype-datetime.html) column, but it is
+also possible to create your own custom scheme for handling
+aggregation for tables that are using an integer time
+column. This can be useful if you have tables that use other measures
+of time that can be represented as integer values, such as nanosecond
+epochs, minutes since founding date, or whatever is suitable for your
+application.
+
+As an example, suppose that you have a table with CPU and disk usage
+for some devices where time is measured in
+[microfortnights][fff-system]. Since you are using an integer-valued
+column as time, you need to provide the chunk time interval when
+creating the hypertable. In this case, let each chunk consist of a
+millifortnight (which is about 20 minutes).
+
+```sql
+CREATE TABLE devices(
+  time BIGINT,        -- Timestamp in microfortnights
+  cpu_usage INTEGER,  -- Total CPU usage
+  disk_usage INTEGER, -- Total disk usage
+  PRIMARY KEY (time)
+);
+
+SELECT create_hypertable('devices', 'time',
+                         chunk_time_interval => 1000);
+```
+
+To define a continuous aggregate on a table that is using an integer
+dimension, it is necessary to have a function to get the current
```

Comment:
I think it is good to mention that it is `time dimension` and `integer` is a type of it.
```suggestion
To define a continuous aggregate on a table that is using an integer
time dimension, it is necessary to have a function to get the current
```
or 
```suggestion
To define a continuous aggregate on a table that is using an integer for
the time dimension, it is necessary to have a function to get the current
```

Reply:
Done.